### DISCUSSION TITLE
Welcome to dlt-viewer Discussions!

### DISCUSSION THREAD

**Discussion by alexmucde** [COVESA/dlt-viewer#229] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sathyan1997** [comment]
Hi,
In DLT viewer we can connect using UDP the IP and Port of the server is given in the Dialog box, After the ECU connect the logs weren't reporting.
The same data over other drivers like UART is reporting in DLT, what is the reason behind the issue?

**Comment by alexmucde** [comment]
@sathyan1997 In the lattest version for UDP you have to select the Ethernet Interface in the dropdown box instead of using the IP address.

**Comment by sathyan1997** [comment]
Hi @alexmucde i am using version 2.20.0 DLT viewer in the drop down of the Interface Type only("UDP","TCP","SERIAL") is available

**Comment by alexmucde** [comment]
@sathyan1997 Yes correctly, select UDP and in the UDP Tab select the receiving interface.

**Reply by sathyan1997** [reply]
okay @alexmucde in the UDP whether i need to configure the Receiving IP and Source Target port or else Any IP and Receiving Port

**Reply by sathyan1997** [reply]
The data transmitted over UDP over the Source port 2xxxx to destination port 5xxxx is visible in Wireshark but the same destination port with any ip is configured in DLT viewer and the data is not received in DLT Viewer(ECU Connect is also not going online ,showing Binding Failed(transmitter IP and port is given))

**Reply by alexmucde** [reply]
I think it is an an issue  on your PC. On our side it is always working. If you find any issue in the source code let us know.

**Comment by leo-lu-niz** [comment]
Hi Team,

I am using DLT viewer 2.20 vision, and I set filter to only receive DEBUG message, but I have no idea why I could not receive DEBUG message?

I could receive other levels， so I guess it is not the connection problem.

Any help or guidance would be awesome.. thanks in advance.

Leo

**Comment by philrob17** [comment]
Le mer. 14 déc. 2022 à 16:09, sathyan1997 ***@***.***> a
écrit :

**Comment by Samuel-John** [comment]
Hi @alexmucde ,
I am using v2.18.0. I wanted to know if this version of dlt-viewer supports UDP protocol.
While adding ECU, the interface type has the option to select UDP but unlike the latest version there is no separate tab for UDP. Instead there is IP.
I have provided the correct IP in Hostname and the correct IP Port.
Yet, the DLT message is not visible in the dlt-viewer.

Can somebody please let me know what is causing this?

Thanks in advance.

**Reply by Samuel-John** [reply]
Answer is available here: https://github.com/COVESA/dlt-viewer/discussions/563
